user_group/views.py

```diff
@@ -66,3 +68,30 @@ def finalize_response(self, request, response, *args, **kwargs):
 
     def get_queryset(self):
         return GroupMembership.get_for(self.request.user)
+
+
+class UserGroupUserSearchViewSet(viewsets.ViewSet):
+    permission_classes = [permissions.IsAuthenticated]
+
+    def list(self, request, version=None):
+        query = request.query_params.get('query', '').strip().lower()
```

Comment:
Instead of `query` let's use `search` as it is used by DRF by default in other APIs.